OperacoesListaSequencial.cpp

```diff
@@ -0,0 +1,459 @@
+#include <stdlib.h>
+#include <iostream>
+#include <new>
+#include <string>
+
+using namespace std;  // para utilizar cout
```

Comment:
this is a code smell, you should use std::cout instead of declaring the namespace at the beginning.